.gitignore

```diff
@@ -6,4 +6,4 @@ dist
 target
 .idea
 version.properties
-
+output-data/data/*triggers.properties
```

Comment:
Do we want to ignore this file? All of the other output data files are deliberately checked into git, even though they're generated and not manually maintained.